package.json

```diff
@@ -43,6 +43,7 @@
     "babel-preset-es2015": "^6.3.13",
     "babel-preset-react": "^6.3.13",
     "babel-preset-react-hmre": "^1.0.1",
+    "babel-preset-stage-1": "^6.16.0",
```

Comment:
transform-class-properties [seems to be part of stage 2 now](http://babeljs.io/docs/plugins/preset-stage-2/), so we can require `babel-preset-stage-2` here.